src/components/Avatar/Avatar.styles.ts

```diff
@@ -32,6 +42,10 @@ export const StyledAvatarRoot = styled.div<{
           return `
             border-radius: 2px;
           `;
+        else if (size === "3xl" || size === "4xl")
+          return `
+            border-radius: 8px;
```

Comment:
Question; should we be using border radius from the `theme` here?

Reply:
changed this :)